### PR TITLE
(Feature) ValidatorMetadata migration

### PR DESCRIPTION
**Pull request by varasev**
- (Mandatory) Description
Migration function in ValidatorMetadata.sol and migration script migrateMetadata.js. Related to https://github.com/poanetwork/poa-network-consensus-contracts/issues/68

- (Mandatory) What is it: (Fix), (Feature) or (Refactor)
(Feature)

### CODE REVIEW

File: scripts/migrateMetadata.js [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+var Writable = require('stream').Writable;
+const EthereumTx = require('ethereumjs-tx');
+const EthereumUtil = require('ethereumjs-util');
+
+web3 = new Web3(new Web3.providers.HttpProvider("https://" + NETWORK + ".poa.network"));
+
+const GAS_PRICE = web3.utils.toWei('1', 'gwei');
+const GAS_LIMIT = 4700000;
+
+const metadataOldAbi = [{"constant": true,"inputs": [],"name": "proxyStorage","outputs": [{"name": "","type": "address"}],"payable": false,"stateMutability": "view","type": "function"},{"constant": true,"inputs": [{"name": "","type": "address"}],"name": "validators","outputs": [{"name": "firstName","type": "bytes32"},{"name": "lastName","type": "bytes32"},{"name": "licenseId","type": "bytes32"},{"name": "fullAddress","type": "string"},{"name": "state","type": "bytes32"},{"name": "zipcode","type": "uint256"},{"name": "expirationDate","type": "uint256"},{"name": "createdDate","type": "uint256"},{"name": "updatedDate","type": "uint256"},{"name": "minThreshold","type": "uint256"}],"payable": false,"stateMutability": "view","type": "function"}];
```
Comment (vbaranov) [inline scripts/migrateMetadata.js]: @varasev why we hardcode ABI here, but don't get ABI from chain-spec repository from the certain commit?

Comment (varasev) [reply scripts/migrateMetadata.js]: @vbaranov migrateMetadata is a one-time operation, so I thought that it's enough to hardcode ABI right in code.

Comment (vbaranov) [reply scripts/migrateMetadata.js]: @varasev then let's add a comment in the script to which commit these ABIs belong

Comment (varasev) [reply scripts/migrateMetadata.js]: @vbaranov ok, done.

**Pull request merged**